inbc-program/README.md

```diff
@@ -300,10 +300,13 @@ inbc aota --app compose --command down --version 1.0 --containertag compose-up
 ### Description
 Load a new configuration file.   This will replace the existing configuration file with the new file.
 
+📝 The configuration file you provide needs to be named *intel_manageability.conf*. If you wish to send with
```

Comment:
is this still true? when I tried a different type of AOTA command, application install, the .tar file is only used to hold the package and the .pem file is held in a central location on the system (so .tar doesn't add value)

Reply:
Fixed.